src/status_im/chat/models/message.cljs

```diff
@@ -475,3 +476,36 @@
 
       :else
       (send-command cofx params))))
+
+(defn custom-send-command-message [whisper-id address asset amount]
+  {:message  nil,
+   :command  {:command      {:description   "Send a payment",
+                             :bot           "transactor",
+                             :color         "#5fc48d",
+                             :ref           ["transactor" :command 83 "send"],
+                             :name          "send",
+                             :params        [{:name "asset", :type "text", :placeholder "Currency"}
```

Comment:
i18n for all params placeholders

Reply:
Actually, let me see if this can be removed. It shouldn't be required for the message